src/cmap/establish/handshake.rs

```diff
@@ -1,99 +0,0 @@
-use bson::{doc, Document};
```

Comment:
This was moved to `handshake/mod.rs`. Unfortunately git doesn't seem to have figured this out...